tools/prepare-static-serving.js

```diff
@@ -0,0 +1,35 @@
+const yaml = require('js-yaml');
+const fs = require('fs');
+
+let doc;
+try {
+  doc = yaml.safeLoad(fs.readFileSync(__dirname + '/../client/flutter/assets/onboarding/iso_countries.en.yaml', 'utf8'));
+  console.log(doc);
+} catch (e) {
+  console.log(e);
+  throw e;
+}
+
+const countries = doc.countries.map((x) => x.alpha_2_code);
+console.log(countries);
+
+const srcPath = __dirname + '/../client/flutter/assets/content_bundles';
+const files = fs.readdirSync(srcPath);
+
+const destDir = __dirname + '/staticContentBuild';
+fs.mkdirSync(destDir);
+
+for (const file of files) {
+  const regex = /^(?<base>.+)\.(?<lang>[^._]+)(?:_(?<country>[^.]+))?\.yaml$/;
+  const m = file.match(regex).groups;
+  console.log(m);
+  const expanded = m.country ? [`${m.base}.${m.lang}_${country}`] : countries.map((c) => `${m.base}.${m.lang}_${c}.yaml`);
```

Comment:
whoops, one more note - we'll need to copy the extension without the country code as well. e.g. there should be a `protect_yourself.en.yaml`. This line is starting to get pretty dense with this suggestion so may want to expand a bit, but thinking:
```suggestion
  const expanded = m.country ? [`${m.base}.${m.lang}_${country}`] : [`${m.base}.${m.lang}.yaml`, ...countries.map((c) => `${m.base}.${m.lang}_${c}.yaml`)];
```

Reply:
The plain .en. file can't be requested by any client, so we don't need this.